### ISSUE TITLE
How to exclude a specific repo?

### ISSUE THREAD

**Issue by evangineer** [morucci/repoxplorer#61]
I thought that this is what `indexer_skip_projects` was and then I realised it had a different scope from what I was looking for.

**Comment by morucci** [comment]
Do not add it under the 'repos' section in  https://github.com/morucci/repoxplorer#define-projects-to-index
If your are using the repoxplorer-github-organization helper to create the yaml file then there is no such option. But in the same way we have '--skip-fork', we could have --skip-repos that would take a list of repository name to not include in the yaml file.

**Comment by evangineer** [comment]
Have been using `repoxplorer-github-organization`.  

So a `--skip-repos` option would definitely be very helpful here.

**Comment by evangineer** [comment]
Had incorporated `repoxplorer-github-organization` into a cronjob.

As a workaround, I'm going to stop running the cronjob for the relevant organzation and edit out the offending repo from the definition file.

Does that make sense?

**Comment by sf-project-io** [comment]
Yes that makes sense but you won't benefit the automatic update if a new
repo appears in your org.

**Comment by evangineer** [comment]
> 
> Yes that makes sense but you won't benefit the automatic update if a new repo appears in your org.
> […](#)

Yep, that's why I referred to it as a workaround.  

Will need to rerun `repoxplorer-github-organization` every so often and edit the resulting definition file until there is a better solution available.

**Comment by evangineer** [comment]
Looks like I'm going to be hacking on `repoxplorer-github-organization` a bit. See #62. 

So I might end up adding `--skip-repos` myself at some point.
